Libplanet/Net/Peer.cs

```diff
@@ -79,7 +94,7 @@ protected Peer(SerializationInfo info, StreamingContext context)
         /// <seealso cref="Swarm.DifferentVersionPeerEncountered"/>
         [IgnoreDuringEquals]
         [Pure]
-        public int AppProtocolVersion { get; }
+        public int? AppProtocolVersion { get; }
```

Comment:
As this breaks the existing interface (IL-wise at least) we should add this to *CHANGES.md*'s *Backward-incompatible interface changes* section.